src/components/IstioWizards/Slider/Boundaries.tsx

```diff
@@ -28,15 +27,6 @@ const Boundaries = props => {
   );
 };
 
-Boundaries.propTypes = {
-  children: PropTypes.array,
-  min: PropTypes.number,
-  max: PropTypes.number,
-  reversed: PropTypes.bool,
-  showBoundaries: PropTypes.bool,
-  slider: PropTypes.object.isRequired
-};
-
```

Comment:
Should we add the types in the props definition? Otherwise the type check won't happen.

Reply:
Done, added. I also changed the components to be class-based as the other ones on the repo.